# Generated by Django 4.1 on 2022-08-25 16:34

import django.core.exceptions
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('home', '0002_alter_register_email_alter_register_name_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='register',
            name='Email',
            field=models.EmailField(error_messages='please enter the gmail', max_length=60, validators=[django.core.exceptions.ValidationError]),
        ),
    ]
